recipes/libffi.recipe

```diff
@@ -20,17 +20,18 @@ FFI_H_UNVERSAL='''\
 
 class Recipe(recipe.Recipe):
     name = 'libffi'
-    version = '3.2.1'
+    version = '3.1.1'
     autoreconf = True
-    commit = '20562ac0427c3578250d04c6e34fb0127d4551cf'
+    commit = 'frida/master'
```

Comment:
I hesitated with that because there was no clear way to ensure that the commit id would be chosen from the "frida" remote, and not "origin".